package.json

```diff
@@ -36,7 +40,7 @@
     "@angular/router": "7.2.14",
     "@blackbaud/auth-client": "2.14.0",
     "@pact-foundation/pact-web": "7.4.0",
-    "@skyux-sdk/builder": "3.6.2",
+    "@skyux-sdk/builder": "3.6.6",
```

Comment:
Some of the SKY UX packages are a few versions behind. Mind running `skyux upgrade` and `skyux install` (you may need to install the migration plugin to gain access to the `skyux upgrade` command, see: https://app.blackbaud.com/skyux-migration-guide/#installing-the-migration-plugin).